src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -1448,8 +1499,6 @@ describe('lib/viewers/BaseViewer', () => {
             expect(combinedOptions.location).to.deep.equal({ locale: 'en-US' });
             expect(combinedOptions.randomOption).to.equal('derp');
             expect(combinedOptions.localizedStrings).to.not.be.undefined;
-            expect(combinedOptions.intl.language).to.equal('en-US');
```

Comment:
I think we'll want more tests for the final output of the options object.

Reply:
I do check in the `createAnnotator()` test  that this gets called with the correct properties. I have added back the check for language as well.